tests/foreman/api/test_lifecycleenvironment.py

```diff
@@ -198,3 +198,26 @@ def test_positive_search_in_org(self):
             {lc_env_.name for lc_env_ in lc_envs},
             {u'Library', lc_env.name},
         )
+
+    @tier2
+    @skip_if_bug_open('bugzilla', 1348727)
+    def test_verify_bugzilla_1348727(self):
```

Comment:
@omaciel in this case, a more descriptive title can fit. But in many other cases, there isn't a good way to sum up a bz in the number of characters we are afforded. I can change this name, but others won't be as easy.